openmls/src/group/core_group/mod.rs

```diff
@@ -803,6 +800,56 @@ impl CoreGroup {
         );
         backend.key_store().delete::<Vec<EncryptionKeyPair>>(&k.0)
     }
+
+    /// Return supported credentials of all members.
```

Comment:
Added methods. These are useful for leaf node validation later. As I see this... maybe I should already make them iterators (as Konrad suggessted in a previous review.)

Reply:
Oh, sorry? Konrad mentioned this in a comment but we havn't opened an issue for that.